dc

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
Aren't we missing default `docker-compose.yml` in `COMPOSE_FILES`?

Reply:
oh yeah we do :)